core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -62,33 +61,38 @@ class SuperAdminController {
     private final SuperAdminService superAdminService;
 
     /**
-     * Controller for create new tariff.
+     * Controller for create new tariff service.
+     *
+     * @param tariffId {@link Long} tariff id.
+     * @param dto      {@link TariffServiceDto} dto for tariff service.
+     * @param uuid     {@link String} employee uuid.
+     * @return {@link GetTariffServiceDto}
      *
-     * @param dto {@link AddServiceDto} dto for service.
-     * @return {@link AddServiceDto}
      * @author Vadym Makitra.
+     * @author Julia Seti
      */
-    @ApiOperation(value = "Create new tariff")
+    @ApiOperation(value = "Create new tariff service")
     @ApiResponses(value = {
-        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = AddServiceDto.class),
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = TariffServiceDto.class),
```

Comment:
I see you [return](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R83) `GetTariffServiceDto`. Why do you use `TariffServiceDto` here

Reply:
fixed